### ISSUE TITLE
ImageView requiring qualified function call for view()

### ISSUE THREAD

**Issue by jebej** [timholy/ImageView.jl#103]
Not sure why this is happening, shouldn't dispatch choose the proper method?

```
julia> view(img)
WARNING: both ImageView and Base export "view"; uses of it in module Main must be qualified
ERROR: UndefVarError: view not defined
```

**Comment by rsrock** [comment]
I think the issue is that ImageView.view [takes](https://github.com/timholy/ImageView.jl/blob/b6fb89265ad716c15983fcb5251a0d3ceb044fef/src/display.jl#L230) any `AbstractArray`, and that Base does as well.

**Comment by timholy** [comment]
ImageView doesn't currently import Base.view (it's only in 0.5+), so the two are in conflict. That's why you need to scope it. Improvements would be accepted though.

**Comment by timholy** [comment]
There's still a deprecation warning needed, but once we can do that this is fixed. It's now called `imshow`.

Closed by #107
